docs/src/parallel_hpc.md

```diff
@@ -0,0 +1,112 @@
+# Parallelism and High Performance Computing (HPC)
+
+One benefit of ensemble methods is their ability to be parallelized. The parallellism occurs outside of the EKP update, and so is not present in the source code. On this page we provide suggestions for parallelization for two types of problems:
+1. Run a parallel loop or map within your current Julia session.
+2. Run a parallel loop through a read/write file interface and workload manager. We provide some utilities to read/write files in a TOML format.
+
+## Case 1: Parallel code within Julia
+Let's look at the simple example in [Lorenz 96 example](@ref Lorenz-example) where we run solver `GModel.run_G_ensemble` which contains the following loop over the `N_ens`-sized ensemble:
+```julia
+for i = 1:N_ens
+    ...
+    g_ens[:, i] = lorenz_forward(settings, lorenz_params[i])
+end
+```
+Each ensemble member `i` runs the Lorenz 96 model with `settings` configuration, and model parameters `lorenz_params[i]`. The runs do not interact with each other, and the user has several options to parallelize.
+
+## Running examples:
+All of the following example cases are covered in [`distributed_lorenz_example.jl`](https://github.com/CliMA/EnsembleKalmanProcesses.jl/blob/main/examples/Lorenz/distributed_lorenz_example.jl). At the top of file uncomment one of the following options
+1. `case = multithread` 
+2. `case = pmap` 
+3. `case = distfor` 
+
+### Multithreading, `@threads`
+
+To parallelize with multithreading, julia must call the file with a prespecified number of threads, e.g, 
+```
+$ julia --project -t 16 distributed_Lorenz_example.jl
```

Comment:
How many threads to we really need for this example? I think `-t 4` should be sufficient.